### PR TITLE
Update free-courses-en.md - Add Laravel 9 Course (changes made)

### PR DESCRIPTION
**Pull request by jlucier-durable**
## What does this PR do?
Add resource

### Description
Youtube playlist for learning Laravel 9, the latest version of Laravel
### Why is this valuable (or not)?
This is a course that goes over Laravel 9, every topic, step by step.
### How do we know it's really free?
It is on Youtube, only need to watch the occasional ads.
### For book lists, is it a book? For course lists, is it a course? etc.
This is a course, a Youtube playlist with 55 videos.
## Checklist:
- [ X ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ X ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ X ] Include author(s) and platform where appropriate.
- [ X ] Put lists in alphabetical order, correct spacing.
- [ X ] Add needed indications (PDF, access notes, under construction).
- [X  ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by jlucier-durable — COMMENTED** [review]
one of my "blank lines" was actually an indent, it has been fixed

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1179,6 +1180,11 @@
 * [Kubernetes Tutorial for Beginners](https://www.youtube.com/playlist?list=PLy7NrYWoggjziYQIDorlXjTvvwweTYoNC) - TechWorld with Nana
 
 
+### Laravel
+
+* [Laravel 9 Tutorial](https://www.youtube.com/playlist?list=PL8p2I9GklV47Jszga434vZxOmY74Q1N_K) - Anil Sidhu (Code Step By Step)
+
+
```
Comment (Thenlie) [inline courses/free-courses-en.md:1187]: We have been putting Laravel courses under the PHP heading. I would move this resource there.

Comment (Thenlie) [reply courses/free-courses-en.md:1187]: Once you have made the requested changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!

**Review by jlucier-durable — COMMENTED** [review]
changes made

**Review by jlucier-durable — COMMENTED** [review]
alphabet mishap

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1331,6 +1325,7 @@
 
 ### PHP
 
+* [Laravel 9 Tutorial](https://www.youtube.com/playlist?list=PL8p2I9GklV47Jszga434vZxOmY74Q1N_K) - Anil Sidhu (Code Step By Step)
```
Comment (Thenlie) [inline courses/free-courses-en.md]: 🤔 I am wondering if the linter should have picked this up and moved it down a line? Not sure if numbers are considered in the alphabetization.

**Comment by Thenlie** [comment]
Other than the comment above, this LGTM.

File: courses/free-courses-en.md (hunk 2 above)
Comment (jlucier-durable) [reply courses/free-courses-en.md]: Yeah, the linted did pick it up and suggested I move it down a line. It is now located under the Laravel 5.something tutorial,

**Review by Thenlie — APPROVED** [review]

**Pull request merged**